cas-server-core-webflow/src/main/java/org/apereo/cas/web/flow/AbstractCasWebflowConfigurer.java

```diff
@@ -163,7 +133,12 @@ protected TransitionableState getStartState(final Flow flow) {
      * @param actions the actions
      * @return the action state
      */
-    protected ActionState createActionState(final Flow flow, final String name, final Action... actions) {
+    @Override
+    public ActionState createActionState(final Flow flow, final String name, final Action... actions) {
+        if (containsFlowState(flow, name)) {
+            logger.debug("Flow {} already contains a definition for state id {}", flow.getId(), name);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String literal "Flow {} already contains a definition for state id {}" appears 5 times in this file; the first occurrence is on line 139](https://www.codacy.com/app/mmoayyed/cas/file/2749420172/issues/source?bid=3256115&fileBranchId=3357337#l139)